scalameta/scalameta/shared/src/test/scala/scala/meta/tests/parsers/XmlSuite.scala

```diff
@@ -236,6 +237,20 @@ class XmlSuite extends ParseSuite with DiffAssertions {
        |  </tr>
        |}""".stripMargin
   )
+  checkOK("<a>{<b>{1}</b>}</a>")
+  checkOK(
+    """val ips = <ips>{
+      |  for {
+      |    field <-ipsList
+      |    JString(ip) <- field.value
+      |  } yield <ip>{ ip }</ip>
+      |}</ips>
+    """.stripMargin)
+
+
+  //FIXME checkOK("<a>{}</a>")
```

Comment:
@olafurpg Not sure how to fix this one. scalac infer `(): Unit` here.

It will parse if I have:
```scala
val ScalaExpr = P( "{" ~ "}" | "{" ~ Block ~ "}" )
```
but then the tokens will not contain the splice:
```
            BOF [0..0) ----> class scala.meta.tokens.Token$BOF
                [0..0) ----> class scala.meta.tokens.Token$Xml$Start
      <a>{}</a> [0..9) ----> class scala.meta.tokens.Token$Xml$Part
                [9..9) ----> class scala.meta.tokens.Token$Xml$End
            EOF [9..9) ----> class scala.meta.tokens.Token$EOF
```